entities/dashboard/staffWidgetsBoard.json

```diff
@@ -0,0 +1,115 @@
+{
+    "type": "object",
+    "properties": {
+      "uid": {
+        "type": "string",
+        "description": "Unique identifier of the staffWidgetsBoard object"
+      },
+      "board_layout": {
+        "type": "string",
+        "description": "The CSS class of the layout. This will determine the layout of the widgets dashboard based on the class.",
+        "enum": [
+          "dashboard"
+        ]
+      },
+      "type": {
+        "type": "string",
+        "description": "the grid frame of the layout, determines the layout frame containing the board",
+        "enum": [
+          "2*2",
+          "3*3",
+          "4*4",
+          "2*3",
+          "3*2",
+          "2*4",
+          "4*2",
+          "3*4",
+          "4*3"
+        ]
+      },
+      "sections": {
+        "description": "An array of dashboard sections. Each section contains a set of properties defining it and a list of widgets that will be populated into the section once it is displayed",
+        "type": "array",
+        "items":{
+          "code_name": {
+            "type": "string",
+            "description": "a code name represent type of the section",
+            "enum": [
```

Comment:
This is the wrong enum and a wrong name.
For a section you should have:
      name: "hero", (no need for enum, its just a name, in the hld you had UID, and you need it if this is a db entity)
      widget_height: 3
      widget_width: 3
      layout_type:enum "horizontal_flex, vertical_flex, grid, fixed

Reply:
Ignore context 'hero', was just an example, the real name and values was discusses and approved by views team.

Reply:
This is different from my requirements. I do not approve it. Let's take it back to views.